spec/models/custom_attribute_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe CustomAttribute do
+  it "return correct field type" do
+    string_custom_attribute = FactoryGirl.create(:custom_attribute, :name => "foo", :value => "string", :resource_type => 'ExtManagementSystem')
```

Comment:
@alongoldboim thanks! To clarify: `FactoryGirl.create` is expensive, so if `FactoryGirl.build` will do it would be better to use that